### ISSUE TITLE
duplicates in users listing - offset not honored in users/details processing

### ISSUE THREAD

**Issue by sylvainfaivre** [nextcloud/server#18503]
### Steps to reproduce
In the `/settings/users` page, some users are duplicated.
Using Firefox's web developer network tab, I see that the first request made to `/ocs/v2.php/cloud/users/details?offset=0&limit=25&search=` replies with the whole user list.
Then, when scrolling to the page end, another request is made to `/ocs/v2.php/cloud/users/details?offset=25&limit=25&search=` which returns the users starting with the offset 25 and going to the end.
Then another one with offset=50 and so on...
So it seems that the `limit`parameter is not used, and some users are listed 2 or more times.

### Server configuration

**Operating system:** Debian Stretch

**Web server:** nginx

**Database:** mysql

**PHP version:** 7.1

**Nextcloud version:** 17.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:** updated

**Where did you install Nextcloud from:** tar.gz from the site

**Signing status:**

No errors have been found.

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.3.0
  - activity: 2.10.1
  - bruteforcesettings: 1.4.0
  - calendar: 1.7.1
  - cloud_federation_api: 1.0.0
  - comments: 1.7.0
  - contacts: 3.1.6
  - dav: 1.13.0
  - federatedfilesharing: 1.7.0
  - federation: 1.7.0
  - files: 1.12.0
  - files_external: 1.8.0
  - files_pdfviewer: 1.6.0
  - files_rightclick: 0.15.1
  - files_sharing: 1.9.0
  - files_trashbin: 1.7.0
  - files_versions: 1.10.0
  - files_videoplayer: 1.6.0
  - firstrunwizard: 2.6.0
  - gallery: 18.4.0
  - logreader: 2.2.0
  - lookup_server_connector: 1.5.0
  - nextcloud_announcements: 1.6.0
  - notifications: 2.5.0
  - oauth2: 1.5.0
  - password_policy: 1.7.0
  - privacy: 1.1.0
  - provisioning_api: 1.7.0
  - recommendations: 0.5.0
  - serverinfo: 1.7.0
  - sharebymail: 1.7.0
  - support: 1.0.1
  - survey_client: 1.5.0
  - systemtags: 1.7.0
  - text: 1.1.1
  - theming: 1.8.0
  - twofactor_backupcodes: 1.6.0
  - updatenotification: 1.7.0
  - viewer: 1.2.0
  - workflowengine: 1.7.0
Disabled:
  - admin_audit
  - encryption
  - user_ldap

```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "17.0.1.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "forcessl": true,
        "loglevel": 0,
        "theme": "",
        "maintenance": false,
        "trusted_domains": [
            "oc.fraudbuster.mobi"
        ],
        "secret": "***REMOVED SENSITIVE VALUE***",
        "share_folder": "Shared",
        "trashbin_retention_obligation": 15,
        "versions_retention_obligation": "15,60",
        "filelocking.enabled": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "logtimezone": "Europe\/Paris",
        "overwrite.cli.url": "https:\/\/oc.fraudbuster.mobi",
        "mysql.utf8mb4": true
    }
}

```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox 71.0

**Operating system:** Ubuntu

### Logs

n/a

**Comment by kesselb** [comment]
Thanks for reporting :+1: 

Will be fixed with 17.0.2: https://github.com/nextcloud/server/issues/17903

**Comment by sylvainfaivre** [comment]
thanks
